test/test_runner.py

```diff
@@ -0,0 +1,36 @@
+"""Validate utils from text module."""
+from subprocess import CompletedProcess
+
+from ansiblelint.runner import Runner
+
+SYNTAX_CHECK_STDERR = """\
+[WARNING]: No inventory was parsed, only implicit localhost is available
+[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'
+ERROR! conflicting action statements: debug, always_run
+
+The error appears to be in '/foo/example.yml': line 47, column 7, but may
+be elsewhere in the file depending on the exact syntax problem.
+
+The offending line appears to be:
+
+
+    - name: always run
+      ^ here
+"""  # noqa: E501
+
+
```

Comment:
We are missing at least a test for the use case where the exit code is not 4. It would be nice to also test stdout stuff, but that part is not that important.